### PR TITLE
Update SidebarNav

### PR DESCRIPTION
**Pull request by tarikhamilton**
The ideal setup should've been a relatively positioned. 

Also, made props to hook into into the layoutWrapper and contentWrapper.

These are needed, if you want to change some styles, such as in the Header Above Example

Before submitting a pull request, please make sure the following is done:

1. Make sure your updates are reflected in both the React and HTML side of MDL.
2. If you've fixed a bug or added code that should be tested, add tests!
3. Make sure the test suite passes (`npm test`).

### CODE REVIEW

File: components/SidebarNav/react.stories.js [hunk 1]
```diff
@@ -1,149 +1,183 @@
...
-    ].map(addKey)
+const bottomMenuItems = [
+  {
+    label: 'Log Out',
+    icon: <span dangerouslySetInnerHTML={{ __html: mSignout }} />,
+  },
+].map(addKey)
+
+const Logo = (
+  <a href="/" tabIndex="0">
```
Comment (josephfusco) [inline components/SidebarNav/react.stories.js]: Curious why we need to set tabIndex here?

File: components/SidebarNav/react.stories.js [hunk 2]
```diff
@@ -1,149 +1,183 @@
...
-      },
+const combatStorybookPaddingStyle = {
+  position: 'fixed',
+  top: 0,
+  left: 0,
+}
+
+const Content = () => (
+  <div style={{ padding: 20 }}>
+    <h1>Page</h1>
```
Comment (josephfusco) [inline components/SidebarNav/react.stories.js]: @tarikhamilton This h1 is valid for the _Default_ story however we would need this to be an h2 for the _Header Above Example_ story.

File: components/SidebarNav/react.stories.js (hunk 1 above)
Comment (tarikhamilton) [reply components/SidebarNav/react.stories.js]: The logo prop accepts a component. It isn't an automatic link, so it needs to be provided, if it is a link.

File: components/SidebarNav/react.stories.js (hunk 2 above)
Comment (tarikhamilton) [reply components/SidebarNav/react.stories.js]: Updating.

**Pull request merged**